test/src/test_encode_context.cpp

```diff
@@ -59,13 +59,6 @@ BOOST_AUTO_TEST_CASE(json_encode_context_should_return_same_address_for_multiple
   BOOST_CHECK(address_0 == address_2);
 }
```

Comment:
I think this test has the same comparing-pointers-across-realloc problem as the one below, it's just that the increment sizes are so small that there's a low chance of a physical reallocation in practice.

Reply:
Done